src/controllers/contract/index.ts

```diff
@@ -233,7 +236,7 @@ export const contractDeployCommand: Command = {
             await account.free();
             account = new Account(account.contract, {
                 client: account.client,
-                address: await account.getAddress(),
+                address: await accountAddress,
```

Comment:
await isn't required here, because accountAddress is string, not Promise

Reply:
done